source/chapter2-devicetree-basics.rst

```diff
@@ -484,6 +484,9 @@ Description:
    (such as a stock ticker symbol), and ``model`` specifies the model
    number.
 
+   The compatible string should consist only of lowercase letters, digits,
+   dashes and at most a single comma. Underscores should not be used.
```

Comment:
I would say something like "a single comma is typically only used following a vendor prefix."

Reply:
OK I updated it, but please feel free to edit it to be clearer.